addons/auth_signup/data/mail_template_data.xml

```diff
@@ -341,5 +341,65 @@
             <field name="auto_delete" eval="True"/>
         </record>
 
+        <!-- Email template for new device login alert -->
+        <record id="alert_new_device_email" model="mail.template">
+            <field name="name">Settings: Alert New Device</field>
+            <field name="model_id" ref="base.model_res_users"/>
+            <field name="subject">New Connection to your Account</field>
+            <field name="email_from">"{{ object.company_id.name }}" &lt;{{ (object.company_id.email or user.email) }}&gt;</field>
+            <field name="email_to">{{ object.email_formatted }}</field>
```

Comment:
Why not setting 'partner_to' to ``{{ object.partner_id.id }}`` ? That way we deal with partners, even if the result is the same.